### PR TITLE
Added Tool info module for "cetfuzz"

### PR DESCRIPTION
**Pull request by Sarathkrishnan**
Tool info for CETFUZZ
    An Automatic Test Suit Generator
    URL: https://zenodo.org/records/10065857

### CODE REVIEW

**Comment by PhilippWendler** [comment]
Please do *not* open a new PR if you have changes, update your existing PR. A new PR makes reviewing and following the discussions difficult and thus causes unnecessary effort for everyone involved.

File: benchexec/tools/cetfuzz.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+from benchexec.tools.sv_benchmarks_util import get_data_model_from_task, ILP32, LP64
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for CETFUZZ
+    An Automatic Test Suit Generator
+    URL: https://zenodo.org/records/10065857
```
Comment (PhilippWendler) [inline benchexec/tools/cetfuzz.py:17]: Is this really the preferred URL for your tool? Don't you have a webpage where people could for example see new versions etc.?

File: benchexec/tools/cetfuzz.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("runTool.py")
+
+    def version(self, executable):
+        return self._version_from_tool(executable, use_stderr=True)
+
+    def program_files(self, executable):
+        return self._program_files_from_executable(
+            executable, self.REQUIRED_PATHS, parent_dir=True
+        )
```
[the remark this answers is not in the document]
Comment (Sarathkrishnan) [reply benchexec/tools/cetfuzz.py:46]: Sorry, It was my mistake,
Could you remove the method.

File: benchexec/tools/cetfuzz.py (hunk 1 above)
Comment (Sarathkrishnan) [reply benchexec/tools/cetfuzz.py:17]: https://gitlab.com/Sarathkrishnan/cetfuzz
It our tool updated the repository.

File: benchexec/tools/cetfuzz.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (PhilippWendler) [reply benchexec/tools/cetfuzz.py:46]: Please update your PR yourself with the new URL and fixed code.

[the remark this answers is not in the document]
Comment (Sarathkrishnan) [reply benchexec/tools/cetfuzz.py]: Yes, Now I updated the request

**Pull request merged**